client-app/constants/Enums.js

```diff
@@ -0,0 +1,9 @@
+const ItemStatus = Object.freeze ({
+    DONATED: 'Donated',
+    IN_STORAGE: 'In storage facility',
+    REFURBISHED: 'Refurbished',
+    SOLD: 'Item sold',
+    RECEIVED: 'Received'
+});
+
+export default ItemStatus;
```

Comment:
This code should be moved to src folder, better this code be removed for code duplicate code.